darwin/Platform.h

```diff
@@ -10,6 +10,7 @@ in the source distribution for its full text.
 
 #include <stdbool.h>
 #include <sys/types.h>
+#include <AvailabilityMacros.h>
```

Comment:
Isn't this only used in `darwin/Platform.c`? If so, it should go there instead as this header is not required to form the public interface of `darwin/Platform.h`.